### ISSUE TITLE
if you have time

### ISSUE THREAD

**Issue by OnlyBGodBoboka** [wac81/keras_triplet_loss#2]
Could you please explain this two line for me

model_q_encoder.compile(optimizer='adam', loss='mse')
model_a_encoder.compile(optimizer='adam', loss='mse')

it seems these two models are not actually trained, is it neccesary to compile them?
Thanks

**Comment by wac81** [comment]
i haven't try it，
model_q and model_a  have same layers to model_trained ,this is a q_encoded = Dense(encode_size)(q_encoded), maybe weights be shared ， i think.
